### PR TITLE
Showing a message instead of including the receipts

### PR DESCRIPTION
**Pull request by pacostas**
## Summary

This PR adds the option instead of showing the receipts, in case they are too large to show a message instead. Avoiding that way a limit error on the size of the release notes file.

## Related issue
* https://github.com/paketo-community/ubi-base-stack/issues/13 close after emerging this PR

## Checklist

* [x] I have viewed, signed, and submitted the Contributor License Agreement.
* [x] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [x] I have added an integration test, if necessary.
* [x] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [x] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

File: actions/stack/release-notes/action.yml
[the remark this answers is not in the document]
Comment (mhdawson) [reply actions/stack/release-notes/action.yml:1]: Does the release notes parse/generate the content? I'm wondering if we couldc have just pass the "To big to display" in through the existing variables like "build_packages_added"

[the remark this answers is not in the document]
Comment (pacostas) [reply actions/stack/release-notes/action.yml:1]: That would be ideal, but In that case it errors as it expects a json array.

[the remark this answers is not in the document]
Comment (pacostas) [reply actions/stack/release-notes/action.yml:1]: Yes, it parses it and specifically iteratese over it in order to print it with a specific format

**Comment by pacostas** [comment]
@mhdawson finished with adding the limit suggestion, I set it to 16 array length as it is approximately something less than 5000 chararacters.

**Review by mhdawson — APPROVED** [review]
LGTM

**Pull request merged**